### PR TITLE
Convert UI timestamps according to timezone

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Output the logs with the correct timezones](https://trello.com/c/tnlsbWhh/723-output-the-logs-with-the-correct-timezones)

File: new-lamassu-admin/src/utils/timezones.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      utcOffsetStr: it.utcOffsetStr,
+      dstOffsetStr: it.dstOffsetStr
+    }),
+    R.uniqBy(
+      it => [it.utcOffset, it.dstOffset, it.utcOffsetStr, it.dstOffsetStr],
+      timezones
+    )
+  )
+
+const getFormattedTimezones = timezones =>
```
Comment (hellypi) [inline new-lamassu-admin/src/utils/timezones.js]: sheesh :D

**Review by hellypi — APPROVED** [review]

File: new-lamassu-admin/src/pages/Locales/Locales.js [hunk 2]
```diff
@@ -121,6 +121,8 @@ const Locales = ({ name: SCREEN_KEY }) => {
   const locale = config && !R.isEmpty(config) ? config : localeDefaults
   const localeOverrides = locale.overrides ?? []
 
+  console.log(data)
+
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Locales/Locales.js]: Console.log

File: new-lamassu-admin/src/pages/Machines/MachineComponents/Details.js [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
 
 import { Label3, P } from 'src/components/typography'
+import { formatDate } from 'src/utils/timezones'
 
 import styles from '../Machines.styles'
 const useStyles = makeStyles(styles)
 
-const Details = ({ data }) => {
+const Details = ({ data, timezone }) => {
+  console.log(timezone)
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Machines/MachineComponents/Details.js]: console.log

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Comment by RafaelTaranto** [comment]
Maybe having a utils for this would help reduce duplicated code

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**